hathor/p2p/manager.py

```diff
@@ -440,12 +447,28 @@ def on_receive_peer(self, peer: PeerId, origin: Optional[ReadyState] = None) ->
         peer = self.received_peer_storage.add_or_merge(peer)
         self.connect_to_if_not_connected(peer, 0)
 
+    def peers_cleanup(self) -> None:
```

Comment:
This function is only called in `reconnect_to_all`, is there no periodic clean up, will peers never expire in the current implementation?

Reply:
I didn't understand. There's a looping call for `reconnect_to_all()` that calls it every 5 seconds.

Reply:
Oh, I missed that. I didn't see that `reconnect_to_all` is called periodically, and I assumed it is only called to close and reopen connections, and not to only try to open new connections like it actually is.